Spellcheck pt. 2 (#6437)
braze-inc/braze-docs: _docs/_partners/message_personalization/dynamic_content/wyng.md

# Wyng

> > [Wyng][0] makes it easy to build interactive digital experiences (i.e. quizzes, preference center, promotions) that engage consumers at the right moments, collect preferences and other zero-party data, and personalize in real time.

Author: ```suggestion
> [Wyng][0] makes it easy to build interactive digital experiences (that is, quizzes, preference center, promotions) that engage consumers at the right moments, collect preferences and other zero-party data, and personalize in real-time.
```
